packages/augur-sdk/src/state/db/TrackedUsers.ts

```diff
@@ -7,9 +7,8 @@ export class TrackedUsers extends AbstractDB {
     super(networkId, networkId + "-TrackedUsers", dbFactory);
   }
 
-  async setUserTracked(user: string): Promise<PouchDB.Core.Response> {
+  async setUserTracked(formattedUser: string): Promise<PouchDB.Core.Response> {
```

Comment:
Does the UI need to specifically add user addresses or does middleware just add them ad the user changes accounts

Reply:
@bthaile UI needs to initialize another sdk/augur with the new address.

Reply:
that is what the UI is doing it destroys old SDK object and creates a new one.